include/triton/core/tritonbackend.h

```diff
@@ -1374,7 +1358,46 @@ TRITONBACKEND_ModelInstanceReportStatistics(
     const uint64_t compute_start_ns, const uint64_t compute_end_ns,
     const uint64_t exec_end_ns);
 
-/// Record statistics for a decoupled inference response.
+/// Create a new decoupled inference response statistics object.
```

Comment:
Is mention of "decoupled" necessary in these docs? I think a non-decoupled backend could choose to publish for a single response if they wanted, right? (though maybe not that useful currently)

Reply:
Sure, removed the word decoupled from comments.